### PR TITLE
fix: Make NNS init .csv parser recognize `dissolve_delay_s` and `maturity_e8s_equivalent` values again

### PR DESCRIPTION
**Pull request by aterga**
This PR fixes the NNS init .csv parser to enable it to recognize `dissolve_delay_s` and `maturity_e8s_equivalent` values for initial neurons again.

### CODE REVIEW

File: rs/nns/governance/init/src/lib.rs [hunk 1]
```diff
@@ -289,29 +294,17 @@ impl GovernanceCanisterInitPayloadBuilder {
...
+            let not_for_profit = record[6]
                 .parse::<bool>()
                 .expect("couldn't read the neuron's not-for-profit flag");
 
-            let memo = if record.len() < 9 {
-                self.rng.next_u64()
-            } else {
-                record[6].parse::<u64>().expect("could not parse memo")
-            };
+            let memo = self.rng.next_u64();
```
Comment (anchpop) [inline rs/nns/governance/init/src/lib.rs:301]: Why remove the ability to specify the memo?

File: rs/nns/governance/init/src/lib.rs [hunk 2]
```diff
@@ -289,29 +294,17 @@ impl GovernanceCanisterInitPayloadBuilder {
...
+                .expect("could not parse maturity_e8s_equivalent");
 
-            let kyc_verified = if record.len() < 11 {
-                false
-            } else {
-                record[8]
-                    .parse::<bool>()
-                    .expect("could not parse kyc_verified")
-            };
+            let kyc_verified = false;
```
Comment (anchpop) [inline rs/nns/governance/init/src/lib.rs:307]: Why remove the ability to specify kyc_verified?

**Review by anchpop — APPROVED** [review]

File: rs/nns/governance/init/src/lib.rs (hunk 2 above)
Comment (aterga) [reply rs/nns/governance/init/src/lib.rs:307]: Despite this code still being here, the ability to specify kyc_verified has been removed in https://sourcegraph.com/github.com/dfinity/ic/-/commit/b0ea17e4028c309ace18a69257268c05fd4b8ef2

File: rs/nns/governance/init/src/lib.rs (hunk 1 above)
Comment (aterga) [reply rs/nns/governance/init/src/lib.rs:301]: Despite this code still being here, the ability to specify memo has been removed in https://github.com/dfinity/ic/commit/b0ea17e4028c309ace18a69257268c05fd4b8ef2

**Pull request merged**